src/cli_plugin/install/download.test.js

```diff
@@ -37,7 +37,7 @@ describe('kibana cli', function() {
       urls: [],
       workingPath: testWorkingPath,
       tempArchiveFile: tempArchiveFilePath,
-      timeout: 0,
+      timeout: 2000,
```

Comment:
@elastic/kibana-operations I did patch version bump (from `2.2.1` to `2.2.4`) for `https-proxy-agent` and `should use https_proxy for secure URLs` test [started to fail with Jest timeout](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/17736/testReport/junit/Jest%20Tests/src_cli_plugin_install/Kibana_Pipeline___kibana_intake_agent___kibana_intake___kibana_cli_plugin_downloader_proxy_support_should_use_https_proxy_for_secure_URLs/), so I had to switch to a finite timeout. 

Does either current or previous behavior raise a red flag for you?

Reply:
@azasypkin I did not spot any reason that would cause that to fail. The only place where that timeout is used is at https://github.com/elastic/kibana/blob/46e7af9c35a1945d1cd0e4cf93b4b37bfb30a176/src/cli_plugin/install/downloaders/http.js#L53 and is passed to `@hapi/wreck`

The only issue I found on `https-proxy-agent` mentioning a problem with jest timeouts was that one https://github.com/TooTallNate/node-https-proxy-agent/issues/75 but it does not look related.